public/index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+
```

Comment:
You can move the `index.html` file to the `html`  folder. The code will be more organized and well structured.